upvsDev/app/app.go

```diff
@@ -0,0 +1,77 @@
+package app
+
+import (
+	"crypto/rsa"
+	"fmt"
+	"net/http"
+
+	muxHandlers "github.com/gorilla/handlers"
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/slovak-egov/einvoice/pkg/handlerutil"
+	"github.com/slovak-egov/einvoice/upvsDev/config"
+)
+
+var corsOptions = []muxHandlers.CORSOption{
+	muxHandlers.AllowedHeaders([]string{"Content-Type", "Origin", "Accept", "X-API-Key", "Authorization"}),
+	muxHandlers.AllowedOrigins([]string{"*"}),
+	muxHandlers.AllowedMethods([]string{"GET", "POST", "OPTIONS", "DELETE", "PUT", "PATCH"}),
+}
+
+type App struct {
+	config          *config.Configuration
+	router          *mux.Router
+	OboTokenPrivate *rsa.PrivateKey
+	OboTokenPublic  *rsa.PublicKey
+	ApiTokenPublic  *rsa.PublicKey
+}
+
+func NewApp() *App {
+	appConfig := config.New()
+
+	a := &App{
+		config:          appConfig,
+		router:          mux.NewRouter(),
+		OboTokenPrivate: getPrivateKey(appConfig.OboTokenPrivateKey),
+		OboTokenPublic:  getPublicKey(appConfig.OboTokenPublicKey),
+		ApiTokenPublic:  getPublicKey(appConfig.ApiTokenPublicKey),
+	}
+
+	a.initializeHandlers()
+
+	return a
+}
+
+func registerHandler(router *mux.Router, method, path string, handler func(http.ResponseWriter, *http.Request) error) {
+	router.HandleFunc(path, handlerutil.ErrorHandler(handler)).Methods(method)
+}
+
+func (a *App) initializeHandlers() {
+	a.router.Use(handlerutil.RequestIdMiddleware)
+	a.router.Use(handlerutil.LoggingMiddleware)
+	a.router.Use(handlerutil.ErrorRecovery)
+
+	registerHandler(a.router, "GET", "/login", a.handleLogin)
+	registerHandler(a.router, "GET", "/logout", a.handleLogout)
+	registerHandler(a.router, "GET", "/api/upvs/user/info", a.handleUserInfo)
```

Comment:
add `/api/sktalk/receive` just returning successful response
Don't bother with validating, this server should really be only for rare cases when dev UPVS does not work so we can continue developing